### PR TITLE
Unify metric names

### LINKED ISSUE #1056: Register standalone metrics without prefix
It looks (I'm not sure 100%, though) that some standalone metrics are registered **with** relay loop prefix, which is an error. By design standalone metrics need to be registered without loop prefix and loop-specific metrics need to use loop prefix. That's because metric names in standalone and complex relays must match. Otherwise it may break some non-alerting widgets in our dashboards (e.g. conversion rate metric).

Having a test for that would be great, however it seems almost impossible without significant efforts.

### PR DESCRIPTION
**Pull request by svyatonik**
closes #1056 

History of this issue:
1) initially we have had separate relay processes for everything and every process has been using 'prefix' feature of [`prometheus::Registry`](https://docs.rs/prometheus/0.13.0/prometheus/struct.Registry.html#method.new_custom). So e.g. headers relay from Rialto to Millau had prefix `Rialto_to_Millau_Sync` and metric names like `best_block_numbers`, resulting in final names like `Rialto_to_Millau_Sync_best_block_numbers` feed to Prometheus;
2) then we've introduced complex relays which have several relays running in single process => we can't use `Registry` prefix anymore. Because different relays have different prefixes - e.g. single process (means single `Registry`) may have relays with prefix `Rialto_to_Millau_Sync` and `Millau_to_Rialto_Sync`. But at the same ideally we should have had metrics with same final names exposed by the process - to avoid using different dashboard for different configurations. That's why prefixes were created directly with final names, without using `Registry` prefix;
3) additionally, we have standalone metrics, which are supposed to be only registered once per process. E.g. Polkadotstorage overhead metric is independent of relay that exposes it.

Right now the situation is as follows:
1) we are still using `Registry` prefix for standalone relays and using final metric names + no-registry-prefix for complex relays;
2) standalone metrics are sometimes exposed several times - e.g. CPU usage may be exposed without prefix (`process_cpu_usage_percentage`) and with prefix (`Rialto_to_Millau_Sync_process_cpu_usage_percentage`).

After this PR, we'll be using final metric names everywhere. We won't be using `Registry` prefix. The only metrics-related thing that'll need fix after this PR is that every relay will keep collecting data for its own copy of standalone metric values, even though only one metric will be exposed. This should be fixed in follow-up PRs.

Things to do here:
- [x] verify dashboards in Rialto <> Millau deployments;
- [x] ~verify Rococo <> Wococo messages relay (since it doesn't use conversion rates, which coming from metrics, which have been altered here)~ checked with altruistic relayer on Rialto <> Millau, because Rococo <> Wococo is too outdated.

### CODE REVIEW

**Comment by svyatonik** [comment]
// takes a bit longer than expected - initially I was going to replace Prometheus registry with our own struct that will essentially be a map of id -> standalone metric. But that was not a best idea, so now I'm moving all standalone metrics to the binary (`sustrate-relay`) level. Loops will only add their specific metrics, which do not need deduplication + values sharing.

**Review by acatangiu — APPROVED** [review]
LGTM

File: deployments/bridges/rialto-millau/dashboard/grafana/rialto-millau-maintenance-dashboard.json [hunk 1]
```diff
@@ -169,14 +169,14 @@
...
-          "expr": "Westend_to_Millau_Sync_kusama_to_base_conversion_rate / Westend_to_Millau_Sync_polkadot_to_base_conversion_rate",
+          "expr": "kusama_to_base_conversion_rate{instance='relay-millau-rialto:9616'} / polkadot_to_base_conversion_rate{instance='relay-millau-rialto:9616'}",
           "interval": "",
           "legendFormat": "Outside of runtime (actually Polkadot -> Kusama)",
           "refId": "A"
         },
         {
           "exemplar": true,
-          "expr": "Rialto_to_Millau_MessageLane_00000000_rialto_millau_to_rialto_conversion_rate",
+          "expr": "Millau_Rialto_to_Millau_conversion_rate{instance='relay-millau-rialto:9616'}",
```
[the remark this answers is not in the document]
Comment (svyatonik) [reply deployments/bridges/rialto-millau/dashboard/grafana/rialto-millau-maintenance-dashboard.json:179]: Need to double check this

[the remark this answers is not in the document]
Comment (svyatonik) [reply deployments/bridges/rialto-millau/dashboard/grafana/rialto-millau-maintenance-dashboard.json:179]: You're right. Expressions were OK - I've fixed widget names.

**Pull request merged**